pkg/virtctl/expand/expand.go

```diff
@@ -0,0 +1,152 @@
+package expand
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	yamlv3 "gopkg.in/yaml.v3"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/tools/clientcmd"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+var (
+	cmd *cobra.Command
+
+	filePathProvided  bool
+	namespaceProvided string
+	outputProvided    string
+)
+
+const (
+	COMMAND_EXPAND = "expand"
+	YAML           = "yaml"
+	JSON           = "json"
+
+	fileFlag, fileFlagShort           = "file", "f"
+	namespaceFlag, namespaceFlagShort = "namespace", "n"
```

Comment:
The `namespace` option is already included in the option for every virtctl commands